integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -272,10 +278,153 @@ context('Incident details (edit) - statement incomplete', () => {
       cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
       const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
       incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
       incidentDetailsPage.submitButton().click()
       cy.location().should(loc => {
         expect(loc.pathname).to.eq(adjudicationUrls.detailsOfOffence.urls.start(34))
       })
     })
+    it('should set DICOVERY button to Yes is DISCOVERY date same as INCIDENT date ', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').should('be.checked')
+    })
```

Comment:
This test isn't testing what the title says - the title refers to yes being selected but it actually asserts that the no radio is selected.
As the date of discovery data isn't included in the stub at the top of the file, the 'no' radio is always going to be selected as there's nothing to compare the incident date to.

Reply:
reworded and set No->Yes